### PR TITLE
Create migration script for UAC-QID-Service

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
We have deprecated the IAC Service, but we need to migrate the data out of the Case Processor and into the `uacqid` schema so that we can guarantee the uniqueness of any UACs and QIDs we generate.

# What has changed
Created migration script.

# How to test?
Set up an environment with the production images of Case Processor and IAC Service. Load an unaddressed batch. Run this script. Check that the data in `uacqid.uac_qid` matches `casev2.uac_qid_link`.

# Links
Trello: https://trello.com/c/mlM35xRH

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[Migrate UAC-QID data from Case Processor into UAC-QID Service in prod](https://trello.com/c/mlM35xRH/940-migrate-uac-qid-data-from-case-processor-into-uac-qid-service-in-prod)

File: manual_scripts/0001-migrate-uac-qid-data.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+-- ***          into the UAC-QID-Service.   ***
+-- *** Author: Nick Grant                   ***
+-- ********************************************
+
+INSERT INTO uacqid.uac_qid (uac, qid, unique_number)
+SELECT uac, qid, unique_number from casev2.uac_qid_link;
+
+SELECT setval('uacqid.uac_qid_unique_number_seq', COALESCE((SELECT last_value FROM casev2.uac_qid_link_unique_number_seq), 1), false);
+
+DROP SCHEMA iac CASCASE ALL;
```
Comment (hbrace) [inline manual_scripts/0001-migrate-uac-qid-data.sql]: CASCADE?

Comment (hbrace) [reply manual_scripts/0001-migrate-uac-qid-data.sql]: Also, the ALL keyword at the end seems to be unnecessary? Datagrip is complaining about it and the IAC schema was only dropped when I removed it.

Comment (NickGrantONS) [reply manual_scripts/0001-migrate-uac-qid-data.sql]: well spotted

**Comment by davidmort** [comment]
theres more work required for this PR as tasked https://trello.com/c/mlM35xRH/940-migration-script-for-uac-qid-data-from-case-processor-into-uac-qid-service-in-prod-5

**Pull request closed without merging**